chapters/alternative-databases/README.md

```diff
@@ -0,0 +1,2 @@
+# Alternative databases
+Now that you've learned how to create and deploy a basic ASP.NET Core application using SQLite, you can explore other databases you can use.
```

Comment:
Let's add a note here explaining that this chapter is optional.

Reply:
Added note indicating optional. This may change if chapter rearranged in future.